services/ui-src/src/libs/formLib.tsx

```diff
@@ -54,11 +54,7 @@ type ConfirmSubmitType = {
 
 export const DefaultFileTypesInfo = () => (
   <p>
-    We accept the following file types:{" "}
-    <b>
-      .doc, .docx, .jpg, .odp, .ods, .odt, .png, .pdf, .ppt, .pptx, .rtf, .txt,
-      .xls, .xlsx, and a few others.
-    </b>{" "}
+    We accept the following file types: <b>.docx, .jpg, .pdf, .png, .xlsx.</b>{" "}
```

Comment:
Though WHOOOPS - missed that the AC says file "formats" not "types" -- probably to keep from being confused with attachment types?!